recipes/skater/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "skater" %}
+{% set version = "1.0.0b10" %}
+{% set md5 = "39c5c7b616badf6f84d437d7c56936ce" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - scipy >=0.15
+    - numpy >=1.10
+    - scikit-learn >=0.18
+    - pandas >=0.19
+    - pathos
+    - dill
+    - requests
+  run:
+    - python
+    - cython
+    - scipy >=0.15
+    - numpy >=1.10
+    - scikit-learn >=0.18
+    - pandas >=0.19
+    - matplotlib
+    - pathos
+    - dill
+    - requests
+
+test:
+  imports:
+    - skater
+
+about:
+  home: https://github.com/datascienceinc/Skater
+  license: MIT
+  license_family: MIT
+  summary: 'Python Library for Model Agnostic Interpretation'
+  description: |
+    Skater is a python package for model agnostic interpretation of predictive
+    models. With Skater, you can unpack the internal mechanics of arbitrary
+    models; as long as you can obtain inputs, and use a function to obtain
+    outputs, you can use Skater to learn about the models internal
+    decision policies.
+  doc_url: https://datascienceinc.github.io/Skater/index.html
+  dev_url: https://github.com/datascienceinc/Skater
+
+extra:
+  recipe-maintainers:
+    - benvandyke
+    - aikramer2
```

Comment:
@aikramer2 -- are you ok with being added as a co-maintainer of the `skater` recipe on conda-forge (you were added by @benvandyke)?